smartsim/settings/lsfSettings.py

```diff
@@ -166,7 +157,27 @@ def set_cpus_per_task(self, cpus_per_task):
         :param cpus_per_task: number of cpus per resource set
         :type cpus_per_task: int
         """
-        self.set_cpus_per_rs(cpus_per_task)
+        self.set_cpus_per_rs(int(cpus_per_task))
+
+    def set_memory_per_rs(self, memory_per_rs):
```

Comment:
@al-rigazzi do you think we should continue to have these aliases? or just implement the specification?

Reply:
@Spartee (cc @MattToast): Answering this and your general comment: I am not really sure we can map everything in LSF to node/task/etc. We can try, and simply say that for everything else, a standard `run_arg` method can be used. 

About `set_walltime` in `AprunSettings`. I cannot think of a reason why we should have it now, I guess it made sense in an old version of the library, possibly.